packages/stark-ui/src/modules/app-logout/components/app-logout.component.scss

```diff
@@ -0,0 +1,8 @@
+stark-app-logout {
+  svg {
+    fill: #fff;
+  }
+  & mat-icon {
+    margin-bottom: 3px;
```

Comment:
True the icon is not aligned vertically, I will create a ticket to fix this in the _button-theme.scss.

So for now you can remove it :-)